setup.cfg

```diff
@@ -4,12 +4,11 @@ ignore =
     W504,  # W504: Line break occurred after a binary operator
     E741  # E741: Do not use variables named 'I', 'O', or 'l'
 exclude = setup.py,
-        asv/commands,
+        asv/benchmarks,
```

Comment:
Why are we reintroducing this to the excludes?

Reply:
Sorry, it was a mistake.